src/utils/hex.cljs

```diff
@@ -25,3 +26,14 @@
   [:=>
    [:cat :string]
    :string])
+
+(defn number-to-hex
+  [value]
+  (->> value
```

Comment:
Similar to `normalize-hex`, could be a good idea to make `number-to-hex` respect nil punning.